### ISSUE TITLE
too obvious, issue only happens here, not the normal hqosu

### ISSUE THREAD

**Issue by osuplayer-hqquestion** [xxCherry/hqosu-legacy#16]
normal hqosu (cant connect online, old version of osu, etc) - https://prnt.sc/q21nem

you can see how the clicks are spread out, looks normal, and this actually helps play the map sometimes because aim doesnt have to be 100% perfect exact timing

on cherry (able to play online) - https://prnt.sc/q21o93

same map, same time of ss, 100% acc, its all clumped and only taps at a specific time

is there any way to fix this

**Comment by MiclySlGa** [comment]
There are some legit configs on mpgh.net
Or, you can just make your own config.

**Comment by osuplayer-hqquestion** [comment]
I used the exact same config for both screenshots

**Comment by MiclySlGa** [comment]
Did  you try using t he  legit configs i told   you  about?

**Comment by osuplayer-hqquestion** [comment]
yes, ive tried using legit configs off mpgh, and even messed around with some configs myself just trying to make that part seem legit, and i can never get it to work

**Comment by MiclySlGa** [comment]
For me it's just working fine, no errors at all. It's probably your Microsoft C++. Try reinstalling your drivers and try booting it up again. If that doesn't work try using the app using compatibility mode.
